### ISSUE TITLE
Music and ui sounds muffled/affected by physics

### ISSUE THREAD

**Issue by CopperGiraffe** [henkelmax/sound-physics-remastered#9]
**Bug description**
The music and ui sounds are muffled/affected by the mod

**Steps to reproduce the issue**
1. Launch a game
2. Wait for music or press pause
3. Click on any button
4. Sound is muffled

**Expected behavior**
The UI sounds and the music should not be affected by this (awesome nonetheless) mod

**Log files**
[Basic logs](https://gist.github.com/CopperGiraffe/34b3f21eb8bcc79f06ff08127e35b05e)
Not sure they will help much

**Versions**
- 1.18.rc-4 (rc1 aswell)
- Fabric version 0.12.5
- Mod version 1.18-rc1-1.0.1

**Other mods**
None

**Comment by henkelmax** [comment]
Yeah, this was an oversight when porting to 1.18. It was actually already fixed but it got lost somewhere when porting.
